src/content/en/2020/fonts.md

```diff
@@ -22,3 +22,266 @@ queries: 04_Fonts
+
+The `unicode-range` feature is a newer approach to this problem. The font is sliced into subsets, each with a separate `@font-face` rule that indicates the Unicode coverage for that slice with a `unicode-range` descriptor. The browser then analyzes the content as part of its rendering pipeline, and downloads *only* the slices needed to render that content.
+
+{{ figure_markup(
+  image="fonts-usage-of-unicode-range.png",
+  caption="usage of unicode-range.",
+  description="A bar chart showing the fraction of unicode-range usage in mobile and desktop web pages that use web fonts.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=2018216759&format=interactive",
+  sheets_gid="655527383",
+  sql_file="04_08.font_unicode_range_with_fcp.sql"
+  )
+}}
+
+Correctly applying `unicode-range` is tricky, as there's a lot of complexity to the way text layout maps Unicode into glyphs, but Google Fonts does this automatically and transparently. It's only likely to be a win for fonts with large glyph counts. In any case, current usage is 37% on desktop and 38% on mobile.
+
+## Formats and MIME types
+
+WOFF2 is the best compression format, and is now [supported](https://caniuse.com/woff2) by effectively all browsers except for versions 11 and earlier of Internet Explorer. It's *almost* possible to serve web fonts using an `@font-face` rule with a WOFF2 source only. This format makes up about 75% of all fonts served.
+
+{{ figure_markup(
+  image="fonts-web-font-mime-types.png",
+  caption="Popular web font MIME types.",
+  description="A bar chart showing the breakdown of percentage of different MIME types for serving web fonts.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=1771540678&format=interactive",
+  sheets_gid="655527383",
+  sql_file="04_10.font_formats.sql"
+  )
+}}
+
+WOFF is an older, less efficient compression mechanism, but almost universally supported, accounting for an additional 11.6% of fonts served. In almost all cases (Internet Explorer 9-11 being the main exception), serving a font as WOFF is leaving performance on the table, and shows a risk of self-hosting. Even if the format choices were optimal at the time of integration, it requires extra effort to update them as browsers improve. Using a hosted service guarantees that the best format is chosen, along with all relevant optimizations.
+
+Ancient versions of Internet Explorer (6-8), which still make about 1.5% of global browser share, require EOT. These don't show up in the top 5 MIME formats, but are necessary for maximum compatibility.
```

Comment:
I think we should be careful about still recommending that EOT is necessary for best compatibility.

Reply:
It seems to me that's what I wrote, but probably that's a sign I wasn't as clear as I should have been. Can you (or someone) suggest better wording?

Reply:
```suggestion
Ancient versions of Internet Explorer (6-8), which still make about 1.5% of global browser share, only support the EOT format. These don't show up in the top 5 MIME formats, but are necessary for maximum compatibility.
```

Maybe leaving out "requires" makes it less of a recommendation and more of an observation?